widget/select.go

```diff
@@ -168,9 +218,12 @@ func (s *Select) Tapped(*fyne.PointEvent) {
 		items = append(items, item)
 	}
 
-	s.popUp = newPopUpMenu(fyne.NewMenu("", items...), c)
-	s.popUp.ShowAtPosition(s.popUpPos())
-	s.popUp.Resize(fyne.NewSize(s.Size().Width, s.popUp.MinSize().Height))
+	s.popUp = ShowPopUpMenuAtPosition(fyne.NewMenu("", items...), c, s.popUpPos())
+	if s.popUp != nil {
```

Comment:
how can an object you just created be nil?

Reply:
Added the test because I had a panic there in earlier code. Now removed.